docs/source/datasets/index.rst

```diff
@@ -60,6 +60,9 @@ The following columns are included in this dataset:
    * - Last name
      - :code:`last_name`
      -
+   * - Year
+     - :code:`year`
+     - Metadata; not affected by noise functions.
```

Comment:
I think I understand what you mean by this, but I wonder if the term "metadata" will make sense to our users in this context?

Also, could we list this before the name columns? Let's check where this actually is in the column order of the pseudopeople output as well. Putting it in the middle seems confusing.

Reply:
> I think I understand what you mean by this, but I wonder if the term "metadata" will make sense to our users in this context?

I was the one who suggested this wording, but I'm not in love with it either. Do you have any better suggestions, @zmbc or @pletale ? We could just say "Not affected by noise functions," but I was trying to explain why, as the reason seems different from our "ground truth" columns.

> Also, could we list this before the name columns? Let's check where this actually is in the column order of the pseudopeople output as well. Putting it in the middle seems confusing.

Hmm, do we know in general whether the column order in the data matches the order they're listed in the pseudopeople docs and/or concept model? I hadn't necessarily been thinking of the column order as part of our design specifications, but maybe it should be. The current pseudopeople data always has year/tax_year as the *last* column.

Reply:
> We could just say "Not affected by noise functions," but I was trying to explain why, as the reason seems different from our "ground truth" columns.

Yep makes sense! Nothing better than "metadata" immediately comes to mind. If we wanted to be more explicit, maybe "metadata that would not be collected directly" or something?

> I hadn't necessarily been thinking of the column order as part of our design specifications, but maybe it should be.

It is! The very first version of pseudopeople had the columns in random order which was very distracting, and we requested that the engineers follow the order in the concept model.

Reply:
For the order, I think last and first are both okay! As long as it's not in between the "normal" attributes.

Reply:
@pletale Ok, let's keep `year` / `tax_year` at the **end** of the datasets as currently implemented, so we should list it as the last column both here in pseudopeople and [in the concept model](https://github.com/ihmeuw/vivarium_research/pull/1296).

For the Notes column, let's use @zmbc's suggestion of being more explicit, so it should say "Metadata that would not be collected directly; not affected by noise functions."